eng/source-build-release-pr.md

```diff
@@ -0,0 +1,5 @@
+<!-- This file is a template for a GitHub pull request.  -->
+<!-- The line prefixed by 'Title:' will be submitted as the title of the PR, and the rest of the file will be submitted as the body. -->
+Title: .NET Source-Build $SDK_VERSION $DATE Updates
```

Comment:
Consider $RELEASE_DATE or $DATE.  When I first saw this, I thought $DATE was the date the PR was created